readme/set-current-user.md

```diff
@@ -0,0 +1,92 @@
+[Back to Main README](https://github.com/simplabs/ember-simple-auth/blob/master/README.md)
+
+## How to Set currentUser Data
+
+Although there are various ways to load currentUser data, this is the canonical
+way to do so. Your authentication route will need to send the id of the user
+account. In this example, it is sent as the user_id.
```

Comment:
This is not actually required. The API could also expose an endpoint `/accounts/me` that returns the current user it identifies based on the authorization header.